retrieval/discovery/kubernetes/types.go

```diff
@@ -13,27 +13,27 @@
 
 package kubernetes
 
```

Comment:
Please undo any changes to this file. It is essentially vendored from a Kubernetes package and modifying it will just cause more work if their API package becomes reasonable to use in general.